syncano_cli/execute/commands.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import sys
+from ConfigParser import NoOptionError
+
+import click
+import json
+from syncano_cli import LOG
+from syncano.models.incentives import ScriptEndpoint
```

Comment:
Also you do not need to import ScriptEndpoint directly, instead use:

```
instance.script_endpoints.get('instance-name', 'script-name')
```

script_endpoints in above sample is a Manager for ScriptEndpoints